SmRtAPI/SmRtAPI/Messages/SetRecognitionConfigMessage.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using Speechmatics.Realtime.Client.Messages;
+
+namespace Speechmatics.Realtime.Client.Messages
+{
+    internal class SetRecognitionConfigMessage : BaseMessage
+    {
+        public override string message => "SetRecognitionConfig";
+        public Dictionary<string, object> transcription_config { get; }
+
+        public SetRecognitionConfigMessage(AdditionalVocabSubMessage additionalVocab = null, 
```

Comment:
One of the slightly odd things about `SetRecognitionConfigMessage` is that it lets you change settings during a stream

https://docs.speechmatics.com/rt-api-ref#setrecognitionconfig

but now we only document that three settings work (`max_delay`, `max_delay_mode`, `enable_partials`) so might be better to stick with what the docs say and do most settings in StartRecognition not here.

iirc the reason I wanted to use this message in the client (before I made it public) is because it often found bugs..

Reply:
(happy to leave this for now if it works)